### ISSUE TITLE
feat [CLI]: `--debug` flag

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1672]
Should print:
- flag values
- WA_ env vars

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.14.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.13.2...3.14.0) (2021-05-16)

- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.22.1 to 4.23.0 [`#1660`](https://github.com/open-wa/wa-automate-nodejs/pull/1660)
- build(deps-dev): bump @types/node from 15.0.2 to 15.0.3 [`#1668`](https://github.com/open-wa/wa-automate-nodejs/pull/1668)
- build(deps-dev): bump @typescript-eslint/parser from 4.22.1 to 4.23.0 [`#1661`](https://github.com/open-wa/wa-automate-nodejs/pull/1661)
- build(deps): bump @types/sharp from 0.28.0 to 0.28.1 [`#1662`](https://github.com/open-wa/wa-automate-nodejs/pull/1662)
- build(deps-dev): bump @grupoboticario/news-fragments [`#1665`](https://github.com/open-wa/wa-automate-nodejs/pull/1665)
- build(deps): bump socket.io from 4.0.2 to 4.1.1 [`#1666`](https://github.com/open-wa/wa-automate-nodejs/pull/1666)
- build(deps): bump rxjs from 7.0.0 to 7.0.1 [`#1664`](https://github.com/open-wa/wa-automate-nodejs/pull/1664)
- 📝  docs: improce message preprocessor documentation [`e9d3547`](https://github.com/open-wa/wa-automate-nodejs/commit/e9d35471aeb51ce443b59e2c3f8a1e813931a4ac)
- Release 3.14.0 [`8f9ee16`](https://github.com/open-wa/wa-automate-nodejs/commit/8f9ee16a8e93eb0fe8a98a0eef5797166c33682a)
- ✨  feat: added `config.messagePreprocessor` #1669 [`d2e652f`](https://github.com/open-wa/wa-automate-nodejs/commit/d2e652f20c0fa7574ef6b798898c8e5bc49ddfb0)
- 🚨  added `without` util, fixed eslint issues in `tools` [`67137b9`](https://github.com/open-wa/wa-automate-nodejs/commit/67137b93e7c2a69f3eb8ce337216b908ba4057b8)
- ✨  feat [CLI]: `--debug` flag, logs all config and WA_* env vars #1672 [`820beef`](https://github.com/open-wa/wa-automate-nodejs/commit/820beef66bfc14b66d79041231a219bce7a6a449)
- 🏷️  types: update `Message` with `filePath` property [`96cbbbb`](https://github.com/open-wa/wa-automate-nodejs/commit/96cbbbb18a7def0616fc936aef776bfa28c025b7)
- 🐛 fix [cli]: `--api-host` causes error during collection creation #1670 [`d78a125`](https://github.com/open-wa/wa-automate-nodejs/commit/d78a125f0b77898872ed27a42a5c5f9ac5db323a)
- ♻️  refactor preprocessors.ts [`37f39ca`](https://github.com/open-wa/wa-automate-nodejs/commit/37f39ca23bc84445e157cdd70cd9a6540117998f)
- ✨  feat: timeout `phoneIsOutOfReach` if it takes longer than 20 seconds #1671 [`47ef8dc`](https://github.com/open-wa/wa-automate-nodejs/commit/47ef8dcb93d2a774e51be21556ee652d6488b710)
- ✨ feat [CLI]: automatically expose `/media` endpoint when `--message-preprocessor="AUTO_DECRYPT_SAVE"` #1669 [`aecc2d1`](https://github.com/open-wa/wa-automate-nodejs/commit/aecc2d17e1cb80017a165dc49c5839c016817a03)
- 🐛  fix [launch]: make sure the navigation error is thrown instead of just logged #1636 [`0a0643c`](https://github.com/open-wa/wa-automate-nodejs/commit/0a0643c4186485958faf677ba7ff583fc5ad530d)
- 🔨  deafults [CLI]: set fallback port to `8080` [`18e9909`](https://github.com/open-wa/wa-automate-nodejs/commit/18e990975ac9f4c0232e85efd391a74bba2b4352)
